.github/workflows/pitest.yml

```diff
@@ -0,0 +1,28 @@
+name: Pitest
```

Comment:
Workflow name and filename is a little bit cryptic.
`mutation-test` would be more self explanatory.